apps/design/backend/src/language_and_audio/speech_synthesizer.ts

```diff
@@ -18,7 +18,14 @@ export const GoogleCloudVoices: Record<
   LanguageCode,
   { languageCode: string; name: string }
 > = {
-  [LanguageCode.CHINESE]: { languageCode: 'cmn-CN', name: 'cmn-CN-Wavenet-B' },
+  [LanguageCode.CHINESE_SIMPLIFIED]: {
+    languageCode: 'cmn-CN',
+    name: 'cmn-CN-Wavenet-B',
+  },
+  [LanguageCode.CHINESE_TRADITIONAL]: {
+    languageCode: 'cmn-CN',
+    name: 'cmn-CN-Wavenet-B',
+  },
   [LanguageCode.ENGLISH]: { languageCode: 'en-US', name: 'en-US-Neural2-J' },
   [LanguageCode.SPANISH]: { languageCode: 'es-US', name: 'es-US-Neural2-B' },
```

Comment:
Thanks - sounds good! Makes sense that we'll need mappings for speech anyway, since the text/audio concerns are slightly different.
And yeah, the world of language tag variations is a rabbit hole that I'm scared to go down 😂